db/initChains.ts

```diff
@@ -2,24 +2,21 @@
 import ArbChain from "./models/Arbchain";
 require("dotenv").config();
 
-(()=>{
-    ArbChain.create({
-        id: 42161,
-        lastBlockChecked: 15210167,
-        l2rpcURL: "https://Arb1-graph.arbitrum.io/rpc",
-        l1rpcURL: "https://mainnet.infura.io/v3/" + process.env.INFURA_ID,
-        inboxAddress: "0x4Dbd4fc535Ac27206064B68FfCf827b0A60BAB3f"
-      })
+export const initChainsProgress = async () => {
+  await ArbChain.create({
+    id: 42161,
+    lastBlockChecked: 15210167,
+    l2rpcURL: process.env.L2_ONE_RPC,
+    l1rpcURL: "https://mainnet.infura.io/v3/" + process.env.INFURA_ID,
+    inboxAddress: "0x4Dbd4fc535Ac27206064B68FfCf827b0A60BAB3f"
+  })
 
 
-      ArbChain.create({
-        id: 42170,
-        lastBlockChecked: 15219858,
-        l2rpcURL: "https://a4ba.arbitrum.io/rpc",
-        l1rpcURL: "https://mainnet.infura.io/v3/" + process.env.INFURA_ID,
-        inboxAddress: "0xc4448b71118c9071bcb9734a0eac55d18a153949"
-      })
-
-
-
-})()
+  await ArbChain.create({
+    id: 42170,
+    lastBlockChecked: 15219858,
+    l2rpcURL: process.env.L2_NOVA_RPC,
+    l1rpcURL: "https://mainnet.infura.io/v3/" + process.env.INFURA_ID,
```

Comment:
Let's make the env param the full `ETHEREUM_L1_RPC` instead (and add that to .env.sample)

Reply:
Made all the args in this file to be set in .env.smaple not only l1rpcURL